pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/BaseMultipleSegmentsConversionExecutor.java

```diff
@@ -64,6 +70,12 @@
       File workingDir)
       throws Exception;
 
+  /**
+   * Returns the segment ZK metadata custom map modifier.
+   */
+  protected abstract SegmentZKMetadataCustomMapModifier getSegmentZKMetadataCustomMapModifier(
```

Comment:
Can we move this to `BaseTaskExecutor`?

We have the same abstract function defined in `BaseSingleSegmentConversionExecutor`. After we move this to `BaseTaskExecutor`, we can remove this from both `BaseSingleSegmentConversionExecutor` and `BaseMultipleSegmentsConversionExecutor`

Reply:
Updated.